reference/7.1/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md

```diff
@@ -469,22 +586,54 @@ developer must implement the code to handle the wildcard input. The wildcards
 supported can vary according to the underlying API or PowerShell provider. For
 more information, see [about_Wildcards](about_Wildcards.md).
 
-### Parameter and variable validation attributes
+## Argument completion attributes
+
+### ArgumentCompleter attribute
+
+The **ArgumentCompleter** attribute allows you to add tab completion values to
+a specific parameter. An **ArgumentCompleter** attribute must be defined for
+each parameter that needs tab completion. Similar to **DynamicParameters**, the
+available values are calculated at runtime when the user presses <kbd>Tab</kbd>
+after the parameter name.
+
+For more information, see
+[about_Functions_Advanced_Argument_Completion](about_Functions_Advanced_Argument_Completion.md#argumentcompleter-attribute).
+
+### ArgumentCompletions attribute
+
+The **ArgumentCompletions** attribute allows you to add tab completion values
+to a specific parameter. An **ArgumentCompletions** attribute must be defined
+for each parameter that needs tab completion. The **ArgumentCompletions**
+attribute is similar to **ValidateSet**. Both attributes take a list of values
+to be presented when the user presses <kbd>Tab</kbd> after the parameter name.
+However, unlike **ValidateSet**, the values are not validated.
+
+This attribute was introduced in PowerShell Core 6.0
+
+For more information, see
+[about_Functions_Advanced_Argument_Completion](about_Functions_Advanced_Argument_Completion.md#argumentcompletions-attribute).
+
+## Parameter and variable validation attributes
 
 Validation attributes direct PowerShell to test the parameter values that users
 submit when they call the advanced function. If the parameter values fail the
 test, an error is generated and the function isn't called. Parameter validation
 is only applied to the input provided and any other values like default values
 are not validated.
 
-You can also use the validation attributes to restrict the values that
-users can specify for variables. When you use a type converter along with a
-validation attribute, the type converter has to be defined before the attribute.
+You can also use the validation attributes to restrict the values that users
+can specify for variables. When you use a type converter along with a
+validation attribute, the type converter has to be defined before the
+attribute.
```

Comment:
Yes, the lines were reflowed for 80 column limits per our standard. We use column limits rather than semantic line breaks. This was existing content that was not formatted to our standard.